sdk/cosmos/sdk/src/test/java/com/azure/data/cosmos/sync/CosmosSyncContainerTest.java

```diff
@@ -0,0 +1,257 @@
+
+import com.azure.data.cosmos.CosmosClientBuilder;
+import com.azure.data.cosmos.CosmosContainerProperties;
+import com.azure.data.cosmos.CosmosContainerRequestOptions;
+import com.azure.data.cosmos.CosmosDatabaseForTest;
+import com.azure.data.cosmos.FeedOptions;
+import com.azure.data.cosmos.FeedResponse;
+import com.azure.data.cosmos.IndexingMode;
+import com.azure.data.cosmos.IndexingPolicy;
+import com.azure.data.cosmos.PartitionKeyDefinition;
+import com.azure.data.cosmos.SqlQuerySpec;
+import com.azure.data.cosmos.rx.TestSuiteBase;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CosmosSyncContainerTest extends TestSuiteBase {
+
+    private String preExistingDatabaseId = CosmosDatabaseForTest.generateId();
+    private List<String> databases = new ArrayList<>();
+    private CosmosSyncClient client;
+    private CosmosSyncDatabase createdDatabase;
+
+    @Factory(dataProvider = "clientBuilders")
+    public CosmosSyncContainerTest(CosmosClientBuilder clientBuilder) {
+        super(clientBuilder);
+    }
+
+    @BeforeClass(groups = {"emulator"}, timeOut = SETUP_TIMEOUT)
+    public void beforeClass() {
+        client = clientBuilder().buildSyncClient();
+        createdDatabase = createSyncDatabase(client, preExistingDatabaseId);
+    }
+
+    @AfterClass(groups = {"emulator"}, timeOut = SHUTDOWN_TIMEOUT, alwaysRun = true)
+    public void afterClass() {
+        safeDeleteSyncDatabase(createdDatabase);
+        for (String dbId : databases) {
+            safeDeleteSyncDatabase(client.getDatabase(dbId));
+        }
+        safeCloseSyncClient(client);
+    }
+
+    private CosmosContainerProperties getCollectionDefinition(String collectionName) {
+        PartitionKeyDefinition partitionKeyDef = new PartitionKeyDefinition();
+        ArrayList<String> paths = new ArrayList<String>();
+        paths.add("/mypk");
+        partitionKeyDef.paths(paths);
+
+        CosmosContainerProperties collectionDefinition = new CosmosContainerProperties(
+            collectionName,
+            partitionKeyDef);
+
+        return collectionDefinition;
+    }
+
+    @Test(groups = { "emulator" }, timeOut = TIMEOUT)
+    public void createContainer() throws Exception {
+        String collectionName = UUID.randomUUID().toString();
+        CosmosContainerProperties containerProperties = getCollectionDefinition(collectionName);
+
+        CosmosSyncContainerResponse containerResponse = createdDatabase.createContainer(containerProperties);
+        validateContainerResponse(containerProperties, containerResponse);
+    }
+
+    @Test(groups = { "emulator" }, timeOut = TIMEOUT)
+    public void createContainer1() throws Exception {
```

Comment:
You have many test methods here:
createContainer1,
createContainer2,
createContainer3, etc

please make the test method names more representitive of what they are doing.
for example:

createContainer1 -> createContainer_1000Throughput

or something like that.

Reply:
renamed